presc/evaluations/conditional_distribution.py

```diff
@@ -138,46 +148,58 @@ class ConditionalDistribution:
 
     model: the ClassificationModel to run the evaluation for
     test_dataset: a Dataset to use for evaluation.
-    config: dict of config options. Available options:
-        `binning`: binning scheme to use for a numerical column, passed to `numpy.histogram`.
-            Can be a fixed number of bins or a string indicating a binning scheme, default: "fd"
-        `common_bins`: should the bins be computed over the entire column and shared
-            across groups (`True`) or computed within each group (`False`), default: True
-        `plot_width_fraction`: width of the bars relative to available space on the plot.
-            Smaller means more space between the bars, default: 1.0
+    config: the main config dict
     """
 
-    def __init__(self, model, test_dataset, config=None):
-        self._config = config
+    def __init__(self, model, test_dataset, config):
+        self._config = config["evaluations"]["conditional_distribution"]
```

Comment:
Definitely prefer passing in the entire config and having the Class/Object/function pulling what it needs